centreon/www/class/centreon-clapi/centreon.Config.Poller.class.php

```diff
@@ -672,7 +656,8 @@ public function sendTrapCfg($pollerId = null)
         if (is_null($pollerId)) {
             throw new CentreonClapiException(self::MISSING_POLLER_ID);
         }
-        $this->testPollerId($pollerId);
+        $pollerId = $this->ensurePollerId($pollerId);
+
         $centreonDir = $this->centreon_path;
         $pearDB = $this->dependencyInjector['configuration_db'];
         $res = $pearDB->query("SELECT snmp_trapd_path_conf FROM nagios_server WHERE id = '" . $pollerId . "'");
```

Comment:
Please correct line 663 using a prepared SQL query.

Reply:
@callapa thanks for the comment, fixed.